sipa/utils/__init__.py

```diff
@@ -82,6 +88,38 @@ def support_hotline_available():
         return False
 
 
+@cached(cache=TTLCache(maxsize=1, ttl=300))
+def meetingcal():
+    url = "https://agdsn.de/cloud/remote.php/dav/public-calendars/bgiQmBstmfzRdMeH?export"
+
+    try:
+        response = requests.get(url, timeout=1)
+    except requests.exceptions.RequestException:
+        return []
+        
+    if response.status_code != 200:
+        return []
+    
+    try:
+        calendar = icalendar.Calendar.from_ical(response.text)
+    except ValueError:
+        return []
```

Comment:
Instead of silently returning `[]`, these issues should be internally reported:
Add 
```python
import logging
logger = logging.getLogger(__name__)
```
at the top of the file, and then emit log messages like this:

```suggestion
    try:
        response = requests.get(url, timeout=1)
    except requests.exceptions.RequestException:
        logger.exception("Error when fetching calendar at %s", url)
        return []

    if response.status_code != 200:
        logger.error("Got unknown status code %s", response.status_code)
        return []

    try:
        calendar = icalendar.Calendar.from_ical(response.text)
    except ValueError:
        logger.exception("Could not parse calendar response %s", response.text)
        return []
```
That way it should be reported to our sentry instance.

Reply:
resolved in 8f9ec85308d3736e16b92be2ec4730e7ae79a12e